ui/admin/tests/acceptance/aliases/update-test.js

```diff
@@ -101,12 +90,12 @@ module('Acceptance | aliases | update', function (hooks) {
 
     urls.alias = `${urls.aliases}/${instances.alias.id}`;
```

Comment:
I am curious if we actually need this? The `urls.alias` assigned in the `beforeEach` looks exactly the same. Same for line 109.

Reply:
I'll comment the line and verify tests pass.